tests/provision/local/test.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+. /usr/share/beakerlib/beakerlib.sh || exit 1
+
+rlJournalStart
+    rlPhaseStartSetup
+        rlRun "export TMT_SCRIPTS_DIR=/var/tmp/helper/scripts"
```

Comment:
Isn't it the default path? I believe we spoke about using a temporary directory, to avoid messing with scripts other tmt invocations might be using.

Reply:
No, the default path defined in this PR is `/var/lib/tmt/helper/scripts`
If you prefer I can place them directly under `/tmp` so they get cleared after reboot

Reply:
I'd recommend using a directory created by `mktemp -d`, it really prevents any possible collision. It's very unlikely, but still possible that two people would run the test and clash over this path... I think it's a good habit to isolate one's work.

Reply:
Updated !